Pod/Core/Example.swift

```diff
@@ -7,13 +7,118 @@
 //
 
 import Foundation
+import XCTest
 
 // Yep, turns out that an example is just a dictionary :)
 
 typealias ExampleTitle = String
-typealias ExampleValue = String
+typealias ExampleValue = Any
```

Comment:
Should `ExampleValue` not be a `ExampleStringRepresentable `?

Reply:
Yes, actually it can now. Done in 5098464